File: Contagem.java
import java.util.HashMap;

public class Contagem {

    public static void main(String[] args) {
        int[] meuArray = {1, 2, 2, 3, 3, 3};

        HashMap<Integer, Integer> contagem = new HashMap<Integer, Integer>();

        // Loop pelo array e contar cada elemento
        for (int i = 0; i < meuArray.length; i++) {
            int elemento = meuArray[i];
            if (contagem.containsKey(elemento)) {
                contagem.put(elemento, contagem.get(elemento) + 1);
            } else {
                contagem.put(elemento, 1);
            }
        }

        // Imprime a contagem de cada elemento
        for (int elemento : contagem.keySet()) {
            System.out.println(elemento + ": " + contagem.get(elemento));
        }
    }
}

File: contagem.js
function contarElementos(array) {
    var contagem = {};
  
    // Loop pelo array e contar cada elemento
    for (var i = 0; i < array.length; i++) {
      var elemento = array[i];
      if (elemento in contagem) {
        contagem[elemento]++;
      } else {
        contagem[elemento] = 1;
      }
    }
  
    return contagem;
  }
  
  // Exemplo de uso
  var meuArray = [1, 2, 2, 3, 3, 3];
  var resultado = contarElementos(meuArray);
  console.log(resultado); // Imprime {1: 1, 2: 2, 3: 3}
  